tortoise/ninja_tortoise.go

```diff
@@ -433,11 +443,11 @@ func (ni *NinjaTortoise) addPatternVote(p votingPattern, view map[types.BlockID]
 		}
 
 		if vp, found = ni.TExplicit[b]; !found {
-			ni.Panic(fmt.Sprintf("block %s from layer %v has no explicit voting, something went wrong ", b.String(), blk.Layer()))
+			ni.Panic(fmt.Sprintf("block %s from layer %v has no explicit voting, something went wrong ", b, blk.Layer()))
```

Comment:
something tells me that printing only `b` will nor parse nicely. Should it be b.Id()?

Reply:
its fine, b has a string method the %s uses it